pango/src/attribute.rs

```diff
@@ -154,6 +154,12 @@ macro_rules! define_attribute_struct {
             }
         }
 
+        impl Into<crate::Attribute> for $rust_type {
```

Comment:
```suggestion
        impl From<$rust_type> for crate::Attribute {
```

Like I told you in the comment ;) Then clippy will also be happy.

Reply:
Agh... fixed